learn/advanced/previous_docs_version.md

```diff
@@ -0,0 +1,84 @@
+# Consulting documentation of previous versions
```

Comment:
We want something shorter and less formal, although my suggestion is perhaps too far to the opposite extreme 😅 

```suggestion
# Accessing old documentation
```

Reply:
"olds docs" is way too informal, IMHO, but you make a good point.

```suggestion
# Accessing previous documentation versions
```